### PR TITLE
Support 0.9

### PR DESCRIPTION
**Pull request by libscott**
* Conditions / Fulfillments becomes inputs/outputs
* Remove CID / FID
* Remove inner "transaction"
* Tx ID as Asset ID
* Remove metadata object

### CODE REVIEW

File: bdb_transaction_cli/cli.py [hunk 1]
```diff
@@ -33,51 +32,49 @@ def generate_keys(name):
...
 
     The CREATE transaction creates a new asset.
     """
-    ffill = Fulfillment(Ed25519Fulfillment(public_key=owner_before),
-                        [owner_before])
-    conditions = [Condition.from_dict(c) for c in listify(conditions)]
-    asset = Asset.from_dict(asset) if asset else None
-    tx = Transaction(Transaction.CREATE, asset, [ffill],
-                     conditions, metadata)
+    input = Input.generate([owner_before])
```
Comment (TimDaub) [inline bdb_transaction_cli/cli.py]: Should we use `input_` here instead?

File: bdb_transaction_cli/cli.py [hunk 2]
```diff
@@ -33,51 +32,49 @@ def generate_keys(name):
...
+    outputs = [Output.from_dict(c) for c in listify(outputs)]
+    tx = Transaction(Transaction.CREATE, {"data": asset_data},
```
Comment (TimDaub) [inline bdb_transaction_cli/cli.py]: `{"data": asset_data}` ==> `{'data': asset_data}`

File: docs/examples.rst [hunk 3]
```diff
@@ -105,54 +99,47 @@ get_asset
...
-       },
-       "version": 1
+                   "uri": "cc:4:20:zL3F_XLRs_snrfmdqSFPqEcu-bu1xF6636oSYpNWvIw:96"
+               },
+               "public_keys": [
+                   "EnE1QD5kBY9Zrsp2Ejsp7W7ZMFAcH75SqR9wz6WrUR15"
+               ]
+           }
+       ],
+       "version": "0.9.0.dev"
```
Comment (TimDaub) [inline docs/examples.rst]: Should we just rename this to `0.9.0` so that we have the CLI ready for the release and don't have to touch it anymore?

**Comment by libscott** [comment]
Ok I realise that this shouldn't be merged until 0.9 server is released on pypi so I'll undo the merge and create a tag.

**Pull request merged**